ci_framework/playbooks/07-deploy-edpm-with-bmaas.yml

```diff
@@ -0,0 +1,46 @@
+- hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Call make_edpm_compute_baremetal target
+      ansible.builtin.include_role:
+        name: 'install_yamls_makes'
+        tasks_from: 'make_edpm_compute_baremetal'
+      vars:
+        make_edpm_compute_baremetal_env:
+          NETWORK_ISOLATION: false
+          OPENSTACK_IMG: "{{ cifmw_operator_build_output['operators']['openstack-operator']['image_catalog'] }}"
+
+    - name: Generate Bash Script to check if compute nodes are provisioned
```

Comment:
>> Is there any particular reason of not using ansible here?

Basically, We need to run `oc wait` command for provisioner and bmh resource for which we would need i.e shell/command module .
We are using the ansible copy module to create `wait_for_bmaas_provisioning.sh` and running this shell script using shell module.
@pablintino for which part you are suggesting to use ansible, could you please elaborate?

Reply:
`oc wait` may be a loop with `retries|until` in plain ansible. That's better than a script imho.

This is especially important since our goal is to get some `oc` module like we have the `ci_make` for logging, reproducer and the rest. The less we diverge, the better.